tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_004_pos.ksh

```diff
@@ -90,6 +90,7 @@ mntp2=$TESTDIR2
 #
 # Create filesystem and clone in the storage pool,  mount them and
 # make the mountpoint busy
+# is_linux 0 if platform Linux, 1 if otherwise
```

Comment:
I don't think this comment is needed..  This convention is used elsewhere in the scripts.